Traceback (most recent call last):
  File "C:\Users\HomePC\interp\sae\src\main.py", line 93, in <module>
    main()
  File "C:\Users\HomePC\interp\sae\src\main.py", line 35, in main
    config = SAEConfig(
             ^^^^^^^^^^
TypeError: SAEConfig.__init__() got an unexpected keyword argument 'lr'
